packages/openid4vc/src/openid4vc-issuer/OpenId4VcIssuerServiceOptions.ts

```diff
@@ -37,6 +38,14 @@ export interface OpenId4VciCreateCredentialOfferOptions {
   baseUri?: string
 
   preAuthorizedCodeFlowConfig: OpenId4VciPreAuthorizedCodeFlowConfig
+
+  /**
+   * Metadata about the issuance, that will be stored in the issuance session record and
+   * passed to the credential request to credential mapper. This can be used to e.g. store an
+   * user identifier so user data can be fetched in the credential mapper, or the actual credential
+   * data.
+   */
+  issuanceMetadata?: Record<string, unknown>
```

Comment:
What is the main difference between this and metadata on a record? Would it be possible to type this or give some info about this?

Reply:
Yeah was thinking about it as well, but as it's part if the core flow i was thinking it deserves it's own property.

The type is completely up to the user, so it's not possible to type it further.

Metadata is usually quite hidden, and i see it more for additional/internal extra data. 

This metadata should be easily accessible by the user of the framework to help them in knowing what credential to issue